### PR TITLE
Pull request for admin panel

### PR DESCRIPTION
**Pull request by skjo0c**

### CODE REVIEW

**Comment by takehiromouri** [comment]
Hey @skjo0c what happened to the user admin panels you created? :D It seems to be not here, is it in your local machine? :)

**Comment by skjo0c** [comment]
@takehiromouri I thought I had pushed it before we went working yesterday :smile: wait. I will push it

File: app/admin/forward_freight.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# permit_params :list, :of, :attributes, :on, :model
+#
+# or
+#
+# permit_params do
+#   permitted = [:permitted, :attributes]
+#   permitted << :other if params[:action] == 'create' && current_user.admin?
+#   permitted
+# end
+  permit_params :email, :password, :password_confirmation
```
Comment (takehiromouri) [inline app/admin/forward_freight.rb]: You're almost finished! :) Let's just set the strong parameters and the form fields like you did in business user and broker :)

Comment (skjo0c) [inline app/admin/forward_freight.rb]: done :)

**Pull request merged**